src/pykiso/lib/auxiliaries/communication_auxiliary.py

```diff
@@ -167,10 +167,15 @@ def receive_message(
 
         msg = response.get("msg")
         remote_id = response.get("remote_id")
+        timestamp = response.get("timestamp")
 
         # stay with the old return type to not making a breaking change
-        if remote_id is not None:
+        if remote_id and timestamp:
+            return (msg, remote_id, timestamp)
```

Comment:
That's still a breaking change, or am I missing something?

Reply:
Please add an additional keyword argument `receive_timestamp: bool = False` so that no breaking change occurs